src/SplitPane.vue

```diff
@@ -38,6 +40,9 @@ function dragMove(e: MouseEvent) {
       : container.value.offsetWidth
     const dp = position - startPosition
     state.split = startSplit + +((dp / totalSize) * 100).toFixed(2)
+    const view = container.value.children[1]
```

Comment:
Get the right-side view by using template ref instead of the DOM API.

Reply:
Done.